eZ/Bundle/PlatformInstallerBundle/src/Installer/DbBasedInstaller.php

```diff
@@ -35,7 +35,7 @@ public function setOutput($output)
     protected function copyConfigurationFile($source, $target)
     {
         $fs = new Filesystem();
-        $fs->copy($source, $target);
+        $fs->copy($source, $target, true);
```

Comment:
I'd maybe add the copy option to the method, as an option argument that defaults to false (it was the default until then).

What do you think ?